sdk-lib/src/main/java/cash/z/ecc/android/sdk/block/CompactBlockProcessor.kt

```diff
@@ -430,7 +430,7 @@ class CompactBlockProcessor internal constructor(
                 network,
                 max(
                     it?.value ?: 0,
-                    lowerBoundHeight.value - 1
```

Comment:
This is the cause of the crash when one set up a new wallet at the sapling activation height. I consider this subtraction as a leftover from the previous refactoring rounds, as we do not want to go below it at this point.